components/tikv_util/src/lib.rs

```diff
@@ -487,6 +485,7 @@ pub fn set_panic_hook(panic_abort: bool, data_dir: &str) {
                 // static variables of RocksDB, which may cause other threads encounter
                 // pure virtual method call. So calling libc::_exit() instead to skip the
                 // cleanup process.
+                libc::fflush(std::ptr::null_mut());
```

Comment:
shall we flush also for the abort case?

Reply:
Seems yes. https://man7.org/linux/man-pages/man3/abort.3.html, since glibc 2.27, abort doesn't flush anymore.